rflx/pyrflx/typevalue.py

```diff
@@ -301,9 +304,12 @@ def assign(self, value: bytes, check: bool = True) -> None:
         self._check_length_of_assigned_value(value)
         self._value = value
 
-    def parse(self, value: Bitstring) -> None:
+    def parse(self, value: Union[Bitstring, bytes]) -> None:
         self._check_length_of_assigned_value(value)
-        self._value = bytes(value)
+        if isinstance(value, bytes):
+            self._value = value
+        else:
+            self._value = bytes(value)
```

Comment:
This could be replaced with `self._value = bytes(value)`. Converting bytes to bytes should be a no-op.

Reply:
Fixed by 829c5c0